resources/inputmaps/apple-media-keys.json

```diff
@@ -3,7 +3,9 @@
   "idmatcher": "AppleMediaKeys.*",
   "mapping":
   {
-    "KEY_PLAY": "play_pause",
+    "KEY_PLAY": "play",
+    "KEY_PAUSE": "pause",
+    "KEY_PLAY_PAUSE": "play_pause",
```

Comment:
Hmm, I think the only user is this: https://github.com/plexinc/plex-media-player/blob/47e2f6e39cb8f76fb7a71cb230fbc56fc6627af9/src/input/apple/InputAppleMediaKeys.mm#L48
so we should be able to change that to `INPUT_PLAY_PAUSE` and be fine.